packages/lib-classifier/src/plugins/tasks/SurveyTask/components/SurveyTaskContainer.spec.js

```diff
@@ -0,0 +1,121 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import { default as Task } from '@plugins/tasks/SurveyTask'
+import { task as mockTask } from '@plugins/tasks/SurveyTask/mock-data'
+import SurveyTask from './SurveyTask'
+import { SurveyTaskContainer } from './SurveyTaskContainer'
+
+describe('SurveyTaskContainer', function () {
+  let wrapper
+  let task = Task.TaskModel.create(mockTask)
+  let annotation = task.defaultAnnotation()
+
+  before(function () {
+    wrapper = shallow(
+      <SurveyTaskContainer
+        annotation={annotation}
+        task={task}
+      />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should initialize with the default state', function () {
+    const mockState = {
+      answers: {},
+      filters: {},
+      selectedChoice: ''
+    }
+    expect(wrapper.state()).to.eql(mockState)
+  })
+
+  it('should render a SurveyTask component', function () {
+    expect(wrapper.find(SurveyTask)).to.have.lengthOf(1)
+  })
+
+  it('should pass the component state and props to the SurveyTask component', function () {
```

Comment:
Realizing this test is flawed, as what's noted is the default definitions for `answers`, `filters`, and `selectedChoice`, so it's not testing if `SurveyTaskContainer` is passing anything to `SurveyTask` other than `task`. Maybe I should `setState` to defined values (like the `mockState` in the following test) then test accordingly?